cmd/minishift/cmd/profile/profile_set.go

```diff
@@ -61,6 +62,28 @@ func runProfile(cmd *cobra.Command, args []string) {
 		}
 	}
 	fmt.Printf("Profile '%s' set as active profile\n", profileName)
+
+	err := removeOcContext(lastActiveProfile)
```

Comment:
This should be done before setting the new oc context.

Reply:
the reson for removing it at last was, if the new context was not able to set we also loose the previous context, which i think would be a bad experience.

Reply:
Nope it wont unless user is running the same profile set command over and over again. Because we do not want the previous context for another profile anyway.